src/client/opamCommands.ml

```diff
@@ -144,7 +144,7 @@ let init =
     `P "Any field from the built-in initial configuration can be overriden \
         through $(i,~/.opamrc), $(i,/etc/opamrc), or a file supplied with \
         $(i,--config). The default configuration for this version of opam is:";
-    `Pre (OpamFile.InitConfig.write_to_string (OpamInitDefaults.init_config));
+    `Pre (OpamFile.InitConfig.write_to_string (OpamInitDefaults.init_config()));
```

Comment:
Oops, forgot this wqs included verbatim here! Since now it can get quite big, and we have `--show-opamrc`, better remove it! :laughing: